tests/Custom/SelectTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use Behat\Mink\Tests\Driver\TestCase;
+
+class SelectTest extends TestCase
```

Comment:
As I've mentioned in #241 the test should go into `/driver-testsuite/tests/Form/SelectTest.php` file and be submitted as Mink repo PR.

For example looking as Zombie driver code I can see, that in there setting value by option text is also not supported.